README.md

```diff
@@ -15,29 +15,30 @@ DefraDB is a user-centric database that prioritizes data ownership, personal pri
 
 Read the documentation on [docs.source.network](https://docs.source.network/).
 
-## Table of Contents
-
-- [Install](#install)
-- [Key Management](#key-management)
-- [Start](#start)
-- [Configuration](#configuration)
-- [External port binding](#external-port-binding)
-- [Add a schema type](#add-a-schema-type)
-- [Create a document instance](#create-a-document-instance)
-- [Query documents](#query-documents)
-- [Obtain document commits](#obtain-document-commits)
-- [DefraDB Query Language (DQL)](#defradb-query-language-dql)
-- [Peer-to-peer data synchronization](#peer-to-peer-data-synchronization)
-  - [Pubsub example](#pubsub-example)
-  - [Collection subscription example](#collection-subscription-example)
-  - [Replicator example](#replicator-example)
-- [Securing the HTTP API with TLS](#securing-the-http-api-with-tls)
-- [Access Control System](#access-control-system)
-- [Supporting CORS](#supporting-cors)
-- [Backing up and restoring](#backing-up-and-restoring)
-- [Community](#community)
-- [Licensing](#licensing)
-- [Contributors](#contributors)
+
+Table of Contents
+=================
```

Comment:
todo: Please keep the same syntax as our other headers :)

Reply:
- [x] done